.github/workflows/release.yml

```diff
@@ -14,6 +14,11 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
+      - name: Check cargo package
```

Comment:
I think this should go in `.github/workflows/rust.yml` so that it runs on PRs too